src/lava/lib/dl/slayer/loss.py

```diff
@@ -210,3 +210,43 @@ def forward(self, input, label):
                 label.flatten(),
                 reduction=self.reduction,
             )
+
+
+class DECOLLELoss(object):
+    """
+    Adapted from https://github.com/nmi-lab/decolle-public
+    Computes the DECOLLE Loss for the model, defined as the sum of the per-layer
+    local pseudo-losses + a regularization term
+    """
+
+    def __init__(self, loss_fn, net, reg=0, reduction='mean'):
+        """'
```

Comment:
Let's follow numpy docstring format and make the description formal. Also, we follow the convention of writing docstring for the constructor with the class docstring.

Reply:
Thank you, I have updated the docstring now